x-pack/filebeat/input/googlepubsub/input.go

```diff
@@ -8,11 +8,11 @@ import (
 	"context"
 	"crypto/sha256"
 	"encoding/hex"
-	"fmt"
-	"runtime"
 	"sync"
 	"time"
 
+	"github.com/elastic/beats/libbeat/common/useragent"
```

Comment:
Move this down to the grouping of github.com/elastic imports.

Reply:
Fixed!